src/kernels/gibbskernel.jl

```diff
@@ -0,0 +1,43 @@
+"""
+    GibbsKernel(x, y)
+
+# Definition
+
+The Gibbs kernel is non-stationary generalisation of the Squared-Exponential
+kernel. The lengthscale parameter ``l`` becomes a function of
+position ``l(x)``.
+
+For a constant function``l(x) = c``, one recovers the standard Squared-Exponential kernel with lengthscale `0.5 c`
```

Comment:
See my comment above:
```suggestion
For a constant function``l(x) = c``, one recovers the standard squared exponential kernel with lengthscale `c`.
```

Reply:
ah ok - thanks for thinking about this and figuring it out, it is a bit confusing :)

Reply:
But no that's not true :) That's why we needed to divide by `sqrt(2)`